sdk/lib/rtl/priv.c

```diff
@@ -387,9 +387,12 @@ RtlReleasePrivilege(IN PVOID ReturnedState)
     else
     {
         /* Otherwise, restore old state */
-        ZwAdjustPrivilegesToken(State->Token, FALSE,
-                                State->OldPrivileges, 0, NULL, NULL);
-
+        Status = ZwAdjustPrivilegesToken(State->Token, FALSE,
+                                         State->OldPrivileges, 0, NULL, NULL);
+        if (!NT_SUCCESS(Status))
+        {
+            RtlRaiseStatus(Status);
```

Comment:
Not sure the `RtlReleasePrivilege` function is expected to raise an exception.

Reply:
It raises just above. 
This PR is about killing the warnings while keeping the existing as it is. If this raises concerns about the existing, then let's merge this PR and open new ones for specific issues, otherwise we will never move forward.